official/utils/arg_parsers/parsers_test.py

```diff
@@ -83,6 +84,21 @@ def test_booleans(self):
     assert namespace.multi_gpu
     assert namespace.use_synthetic_data
 
+  def test_parse_dtype_info(self):
+    parser = TestParser()
+    for dtype_str, tf_dtype, loss_scale in [["fp16", tf.float16, 128],
+                                            ["fp32", tf.float32, 1]]:
+      args = parser.parse_args(["--dtype", dtype_str])
+      parsers.parse_dtype_info(args)
+
+      assert args.dtype == tf_dtype
+      assert args.loss_scale == loss_scale
+
+      args = parser.parse_args(["--dtype", dtype_str, "--loss_scale", "5"])
+      parsers.parse_dtype_info(args)
+
+      assert args.loss_scale == 5
```

Comment:
Test that int8/invalid types raise errors, given the discrepancy currently in choices versus this function.